content/shmem_pe_accessible.tex

```diff
@@ -26,9 +26,9 @@
     \FUNC{shmem\_pe\_accessible} routine returns \CONST{TRUE} only if  the  remote
     \ac{PE} is a process  running from the same executable  file as the local
     \ac{PE}, indicating that full \openshmem support for symmetric data objects
-    (that reside in the static memory and symmetric heap) is available, otherwise it
+    that reside in the static memory and symmetric heap is available; otherwise, it
```

Comment:
The text "that reside in the static memory and symmetric heap" actually is a nonessential clause that could remain offset, since it effectively repeats the definition of a symmetric data object.